### ISSUE TITLE
Any information about how to launch installed firefox extension

### ISSUE THREAD

**Issue by prikhi** [prikhi/pencil#424]
_From [anatoly....@gmail.com](https://code.google.com/u/115846696075841189927/) on March 13, 2012 02:36:47_

What steps will reproduce the problem? 1. google pencile project
2. click "download"
3. install extension
4. restart firefox
5. where is my pencil? wtf? i see it in 'extensions' panel installed but how to use it?
6. oh, googling saved me — i need to use 'alt' key, go to tools and finaly launch pencil What is the expected output? What do you see instead? Any page with information on hot to launch Pencil after restarting firefox What version of the product are you using? On what operating system? pencil 1.3.4 in firefox 10 on windows 7 Please provide any additional information below. it's because firefox on win7 don't show menu, instead there is one button 'firefox' but there is no tools

_Original issue: http://code.google.com/p/evoluspencil/issues/detail?id=424_

**Comment by alexandreleroux** [comment]
True, at the moment, we need Alt=>Tools=>Pencil Sketching... to launch it. My suggestion would be to have the same 'Pencil Sketching' item in the 'hamburger' menu under the 'Developer' tab (because I think that's possibly the only place where add-ons can add items). Thanks! :-)

**Comment by prikhi** [comment]
Actually, this should be fixed since #671. Do you not see the `Pencil Sketching` icon under the `Customize` menu?

**Comment by alexandreleroux** [comment]
Thanks @prikhi - you are right, Pencil is not in the hamburger menu by default but can be added by customizing the menu. Would be great if we could make it more obvious to users how to launch Pencil after installation. The current requirement of using Alt=>Tools=>Pencil Sketching or customizing the hamburger menu is not optimal. That said, I'm not certain what flexibility we have to add it by default or elsewhere.

Another possibility would be to open a new tab after initial installation that displays basic instructions, such as how to launch Pencil. I know other plugins do that (i.e. open a website after plugin installation or update), but I don't know how they do it.

Thanks for the great plugin! :-) -- Alex
